src/MapViewController.swift

```diff
@@ -76,7 +76,8 @@ open class MapViewController: UIViewController, LocationManagerDelegate {
   var currentRouteMarker: TGMapMarkerId?
   open var shouldFollowCurrentLocation = false
   open var findMeButton = UIButton(type: .custom)
-  open var currentAnnotations: [PeliasMapkitAnnotation : TGMapMarkerId] = Dictionary()
+  open var annotationToMarkerId: [PeliasMapkitAnnotation : TGMapMarkerId] = Dictionary()
```

Comment:
Reviewed the dictionary extension usage - looks good. 2 Things:

1) Add unit tests for it to make sure it keeps behaving (especially in future swift versions. 4 is on the horizon!) .
2) `annotationToMarkerId` no longer makes a whole lot of sense I think. Any issue with putting it back to its original name?